frontend/src/app/features/work-packages/components/wp-buttons/wp-status-button/wp-status-button.component.sass

```diff
@@ -11,4 +11,7 @@
     max-width: 100%
 
   &--text
-    margin: 0 5px
+    @include text-shortener
+
+  &--button
```

Comment:
There is already a `&--button` selector in this file. So please move the `display: flex` rule to this block as well.